src/controller/CHIPDeviceControllerFactory.h

```diff
@@ -69,6 +69,18 @@ struct SetupParams
     // The Device Pairing Delegated used to initialize a Commissioner
     DevicePairingDelegate * pairingDelegate = nullptr;
 
+    /**
+     * Controls where we permit multiple DeviceController instances to exist
+     * on the same logical fabric (identified by the tuple of the fabric's
+     * root public key + fabric id).
+     *
+     * Each controller instance will be associated with its own FabricIndex.
+     * This pivots the FabricTable to tracking identities instead of fabrics,
+     * represented by FabricInfo instances that can have colliding logical fabrics.
+     *
```

Comment:
It is odd to have a changelog-like comment in the code.  This should just describe what the code currently does, not how it's been changed.

Reply:
So, I meant 'pivot' as not so much 'this CL is changing that concept' but rather, at run-time, this is literally altering the meaning of a FabricInfo instance if this flag is set to True.

Suggestions welcome.

Reply:
Ah, I see.  That makes sense.